### ISSUE TITLE
CITRA RAPIDFIRE / TURBO BUTTON

### ISSUE THREAD

**Issue by KingLeonidas2020** [citra-emu/citra#5484]
I only like to suggest in your great program that Rapidfire / Turbo button is very essential in our gameplay most especially in a PC and keyboard user like me. it can help us play games with ease because with Rapidfire, we can push buttons rapidly with ease. Say, in some games like Monster Hunter requires pushing of buttons rapidly and could make our hands and fingers ache. PPSSPP have rapidfire button so i believe that Citra can make a rapidfire button in their program too. Thank you very much, this is only a suggestion for improvement, your emulator is great and the graphics is so advance.

**Comment by KingGamingYT** [comment]
You opened this as an issue, not a feature request.

**Comment by zhaowenlan1779** [comment]
@KingGamingYT That doesn't really matter.

**Comment by FearlessTobi** [comment]
Isn't this beheavior achievable with many external programs already?

**Comment by KingLeonidas2020** [comment]
yes sir, ppsspp have a turbo / rapid fire button,citra is made by genius
people, i believe they could put it to citra too.. i also hope citra will
be compatible with the latest versions of mobile phones so that more people
can play 3ds games.

Arben G. Montellano
Mechanical Engineer
NW Steel Technologies Inc.
Ochoa St. Tinejero, Pulilan Bulacan 3005
